apis/flowcollector/v1beta1/flowcollector_types.go

```diff
@@ -690,16 +693,23 @@ const (
 	FLPFilterTargetExporters FLPFilterTarget = "Exporters"
 )
 
-// `FLPFilterSet` defines the desired configuration for FLP-based filtering satisfying all conditions
-type FLPFilterSet struct {
-	// `filters` is a list of matches that must be all satisfied in order to remove a flow.
+// `FLPFilters` defines the desired configuration for FLP-based filtering. Flows that match those filters are kept, all other flows are discarded.
+type FLPFilters struct {
+	// `anyOf` contains a list of rules evaluated individually: any satisfied rule results in a match.
 	// +optional
-	AllOf []FLPSingleFilter `json:"allOf"`
+	AnyOf []FLPFilterAnyOf `json:"anyOf"`
```

Comment:
Isn´t this a breaking change ?
In 1.8.0 we have a Allof directly inside the Filter field.

Assuming someone already has some Filters configured, will an upgrade succeed ?